### PR TITLE
Contact form branch

### PR DESCRIPTION
**Pull request by tsheporamantso**
**On this milestone**

- [ ] Added footer section with contact form
- [ ] Layout HTML elements using Flexbox
- [ ] Ensured that GitHub flow is adhered to and also ensured HTML and CSS best practices are followed.
- [ ] Modified README file with contact information of the second author

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,269 +1,336 @@
...
-    </div>
-  </section>
-</body>
+    </section>
+
+    <!-- /**
+        *! Footer Section Here.
+        **/ -->
+
+    <footer class="footer__section">
```
Comment (zieeco) [inline index.html]: - Hey, please, the container for this section is a `<section>` and not a `<footer>`. please, kindly update this as this was stated as footer in the project but it is not really the footer. let's fix this to improve our semantic usage.

File: style.css [hunk 2]
```diff
@@ -469,3 +470,126 @@
...
+  align-items: center;
+  justify-content: center;
+  background-image: url("./images/contact form shapes-mobile.svg");
+  background-repeat: no-repeat;
+  background-position: top right;
+  background-color: #6070ff;
+  border-radius: 70px 0 0 0;
+  margin-top: 100px;
+  height: 779px;
+}
```
Comment (zieeco) [inline style.css]: - Hey, please, let's kindly add the missing black line at the bottom of this section to match the design as much as we can. See the images below:


*your edited design*
Screenshot from 2023-02-21 10-59-56

*design template*
Screenshot from 2023-02-21 10-59-09

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @mercymugambi @tsheporamantso , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linter checks :heavy_check_mark: 
pair-programming :heavy_check_mark: 
form validation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**